lib/pure/memfiles.nim

```diff
@@ -15,6 +15,27 @@
 ## It also provides some fast iterators over lines in text files (or
 ## other "line-like", variable length, delimited records).
 
+runnableExamples("-r:off"):
+  # Example of how to create a memfile and
+  # write data to it and read it again. 
+  import memfiles, os
+
+  let fn = "/tmp/test.mmap"
```

Comment:
```suggestion
  let fn = "test.mmap"
```

That way it's more portable

Reply:
@Yardanico, your changes are fine for me, thanks a lot.

Reply:
use this instead:
```nim
let fn = getTempDir() / "test.mmap"
```